internal/commands/waypoint/add-ons/create.go

```diff
@@ -58,8 +58,8 @@ $ hcp waypoint add-ons create -n=my-addon -a=my-application -d=my-addon-definiti
 					Required:     true,
 				},
 				{
-					Name:         "application-name",
-					DisplayValue: "NAME",
+					Name:         "app",
```

Comment:
I wonder since this is a breaking change, how about we add both into the mix for some period of time, and then let users drift away. Not sure if it is possible, but when specified both could update the same underlying var.

Reply:
I don't feel like that's worth the maintenance effort. Maybe if this CLI had been around for years sure, but at this point it's brand new (not to mention Waypoint is still in beta) so I'd rather make the breaking change than maintain two flags for backwards compatibility right now. I do appreciate the suggestion though!